venafi/provider.go

```diff
@@ -25,14 +25,14 @@ const (
 func Provider() terraform.ResourceProvider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
-			"url": &schema.Schema{
```

Comment:
why the "&schema.Schema" references are being removed  on the resource attributes?